scalafmt-cli/src/main/scala/org/scalafmt/cli/Cli.scala

```diff
@@ -164,10 +164,7 @@ object Cli {
     }
 
     val sbtOptions = options.copy(
-      config = options.config.copy(
-        runner = options.config.runner.copy(
-          dialect = Sbt0137
-        )))
+      config = options.config.copy(runner = options.config.runner.forSbt))
```

Comment:
cc/ @pauldraper You might want to use `ScalafmtRunner.forSbt` instead of `dialects.Sbt0137` in ScalafmtFactory. The former will re-use the existing configured dialect while the latter is Scala211 for sbt files.